base/applications/games/solitaire/solundo.cpp

```diff
@@ -0,0 +1,153 @@
+#include "solitaire.h"
+
+static int source_id = -1;
```

Comment:
Add more comments here and below.
Especially the meaning of minus one.

Reply:
Done.